src/sequence/mod.rs

```diff
@@ -206,17 +206,17 @@ impl Data {
         })
     }
 
-    /// Appends new entry.
-    pub fn append(&mut self, entry: Entry) -> Result<DataWriteOp<Entry>> {
+    /// Generate unsigned crdt op, adding the new entry.
+    pub fn create_append_op(&mut self, entry: Entry) -> Result<DataWriteOp<Entry>> {
```

Comment:
We'll have to make a decision here, do we still want this function to apply the op to the replica or to only generate the operation? 
Currently `LSeq` applies it, thus `create_append_op` not only creates it but also applies it, which may fail as soon as we add signature verification logic in the `apply_data_op` function?

Reply:
Probably it's ok, as we don't want/need any signature verification for applying it to local replica??

Reply:
Hmmm, I was assuming we'd want it to not apply it. Seems cleaner? But also bring lseq in line with other crdt's in how they generate ops.

But you're right in that it's probably not _necessary_ for local data.

Reply:
Perhaps even for local replica we need the ops signed as for the event of a complete state replication, or e.g. when we will be sending old ops to other replica which is not causally ready?

Reply:
Yeh true. Where we save ops (for versioning, resending), will have to be at the `apply` stage (as no other point makes sense).

So yeh, will have to be signed. May be trivial to start saving ops there too now, and then as/when we want we can expose some kind of version api, with `version number` corresponding to the number of ops to apply (naiively at least... I suppose there's a Q of ordering a vec there too 🤔 )